src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodGroup.cs

```diff
@@ -42,17 +41,16 @@ internal void PopulateWithMethods(
             BoundExpression receiverOpt,
             ArrayBuilder<Symbol> members,
             ImmutableArray<TypeWithAnnotations> typeArguments,
-            bool isExtensionMethodGroup,
             LookupResultKind resultKind = LookupResultKind.Viable,
             DiagnosticInfo error = null)
         {
             this.PopulateHelper(receiverOpt, resultKind, error);
-            this.IsExtensionMethodGroup = isExtensionMethodGroup;
+            this.IsExtensionMethodGroup = true;
```

Comment:
>true [](http://example.com/codeflow?start=42&length=4)

Since we use this method only for extensions, consider reflecting this in method's name